File: swagger/models/get_products_response.py
from .base_model import BaseModel

from .pagination_response import PaginationResponse

class GetProductsResponse(BaseModel):
	"""
	no description available

	Attributes:

		pagination_response
		Contains information about the pagination used.

		products
		Contains information about the products.
	"""

	input_map = {
		'PaginationResponse': ('pagination_response', PaginationResponse),
		'Products': 'products',
		}

	output_map = {
		'pagination_response': 'PaginationResponse',
		'products': 'Products',
		}

	pagination_response = None
	products = None

File: swagger/Models/GetProductsResponse.php
<?php

namespace App\MindBody\Models;

/**
 * no description available
 */
class GetProductsResponse extends BaseModel
{
	/**
	 * Contains information about the pagination used.
	 * 
	 * @var PaginationResponse 
	 */
	public $paginationResponse;
	/**
	 * Contains information about the products.
	 * 
	 * @var Product[] 
	 */
	public $products;

	protected function getInputMap()
	{
		return [
			'PaginationResponse' => ['paginationResponse', PaginationResponse::class, null],
			'Products' => ['products', 'array', Product::class],
			];
	}

	protected function getOutputMap()
	{
		return [
			'paginationResponse' => ['PaginationResponse', PaginationResponse::class, null],
			'products' => ['Products', 'array', Product::class],
			];
	}
}
